src/stores/FileBrowserStore/FileBrowserStore.ts

```diff
@@ -568,12 +568,12 @@ export class FileBrowserStore {
         }
     };
 
-    @action setSaveSpectralRangeMin = (min: string) => {
-        this.saveSpectralRange[0] = min;
+    @action setSaveSpectralStart = (start: string) => {
+        this.saveSpectralStart = start;
```

Comment:
can we save saveSpectralStart and saveSpectralEnd as float instead of string so we don't need to parseFloat everytime we try to use them?

Reply:
Thanks for pointing out that. I agree that setting saveSpectralStart/End as "string" is wired too. But, actually, I had changed their types back from number to string in a previous commit(d14ab2e) since the string type would cause a different checking behavior in the input fields.

> @TienHao not sure if this is due to the fix here, the input field becomes more aggressive in checking values. When it is 0 and we hit the backspace key, it stays as 0 but if when we see 0 and we enter 10, for example, it works. In other cases, if it is NaN, there is no way back. I would prefer we keep the original way for input fields. 
> 
> ![Screenshot 2023-09-22 at 12 32 35](https://github.com/CARTAvis/carta-frontend/assets/20819712/6dbc111a-65ae-4cc5-9394-6973e648c7eb)

I am trying whether this can be fixed by also modifying `NumericInput` to `SafeNumericInput` in the ImageSaveComponent to have a proper value checking behavior, since this is the only place left to implement `NumericInput` directly instead of `SafeNumericInput` for inputs.